Templates/colors.json

```diff
@@ -0,0 +1,4687 @@
+      ]
+    },
+    "ObjChaserBrokenBlockSpawner": {
+      "fields": [
+      ]
+    },
+    "ObjColorFilterSpawner": {
+      "fields": [
+        { "name": "CollisionWidth", "type": "float32" },
+        { "name": "CollisionHeight", "type": "float32" },
+        { "name": "FadeSpeed", "type": "float32" },
+        { "name": "FadeTime", "type": "float32" },
+        { "name": "TopFadeSpeed", "type": "float32" }
+      ]
+    },
+    "ObjDenergySpawner": {
+      "fields": [
+        { "name": "FirstSpeed", "type": "float32" },
+        { "name": "SecondSpeed", "type": "float32" }
+      ]
+    },
+    "ObjFloatingFloorSpawner": {
+      "fields": [
+        { "name": "Type", "type": "ObjFloatingFloorSpawner::Type" },
+        { "name": "MinSpeed", "type": "float32" },
+        { "name": "MaxSpeed", "type": "float32" },
+        { "name": "HideEffect", "type": "bool" }
+      ]
+    },
+    "ObjGoalCollisionSpawner": {
+      "fields": [
+        { "name": "CollisionWidth", "type": "float32" },
+        { "name": "CollisionHeight", "type": "float32" }
+      ]
+    },
+    "ObjCleanKillSpawner": {
+      "fields": [
+        { "name": "TargetId", "type": "ObjCleanKillSpawner::TargetId" },
+        { "name": "TargetList", "type": "array", "subtype": "object_reference" },
+        { "name": "LimitTime", "type": "float32" }
+      ]
+    },
+    "ObjEventSetterSpawner": {
+      "fields": [
+        { "name": "TargetList", "type": "array", "subtype": "object_reference" },
+        { "name": "Trigger", "type": "ObjEventSetterSpawner::Trigger" },
+        { "name": "Condition", "type": "ObjEventSetterSpawner::Condition" },
+        { "name": "TimesType", "type": "ObjEventSetterSpawner::TimesType" },
+        { "name": "EventList", "type": "array", "subtype": "object_reference" },
+        { "name": "EventNotifier", "type": "ObjEventSetterSpawner::EventNotifier" },
+        { "name": "EventTimer", "type": "float32" },
+        { "name": "EventList2", "type": "array", "subtype": "object_reference" },
+        { "name": "EventNotifier2", "type": "ObjEventSetterSpawner::EventNotifier2" },
+        { "name": "EventTimer2", "type": "float32" },
+        { "name": "EventList3", "type": "array", "subtype": "object_reference" },
+        { "name": "EventNotifier3", "type": "ObjEventSetterSpawner::EventNotifier3" },
+        { "name": "EventTimer3", "type": "float32" }
+      ]
+    },
+    "ObjHintRingSpawner": {
+      "fields": [
+        { "name": "HintID", "type": "uint32" },
+        { "name": "IsMessageBoot", "type": "bool" },
+        { "name": "OnlyOneBoot", "type": "bool" }
+      ]
+    },
+    "ObjHintRingMultiSpawner": {
+      "fields": [
+        { "name": "HintID", "type": "uint32" },
+        { "name": "HintID2", "type": "uint32" },
+        { "name": "HintID3", "type": "uint32" },
+        { "name": "IsMessageBoot", "type": "bool" },
+        { "name": "OnlyOneBoot", "type": "bool" }
+      ]
+    },
```

Comment:
Parameters named based on in-game functionality and parameter names from other objects with similar functionality. Multiple HintID parameters named based on other objects, such as `SwitchCollision`. Set data parser function (0x8021F84C, NTSC-J Wii) and `ProcessMessage` function (0x800EEDEC, NTSC-J Wii) used for additional help.